crashsimilarity/cache.py

```diff
@@ -0,0 +1,89 @@
+import json
+import pickle
+from abc import abstractmethod
+
+import logging
+
+from crashsimilarity import utils
+
+
+class BaseCache(object):
+    def __init__(self, name, file_name=None):
```

Comment:
But this option have a default value, so code user can just ignore it. The one cool idea in python is to have as many arguments as we want, even if we don't needed every time. 
Moreover, this code is pretty straightforward and the usecase I add this parameter for is to have different `TracesCache`s for different models and\or for different products, saved in different files simultaneously